src/pages/index.html

```diff
@@ -2,29 +2,65 @@
   <head>
     <title>Tuesday's Tunes</title>
 
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="Blasting tunes to you straight from the cloud." />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="Blasting tunes to you straight from the cloud."
+    />
     <meta property="og:title" content="Tuesday's Tunes" />
     <meta property="og:type" content="website" />
     <meta property="og:url" content="http://www.tuesdaystunes.tv/" />
-    <meta property="og:image" content="http://www.tuesdaystunes.tv/assets/tuesdays-tunes.webp">
-    
-    <link rel="stylesheet" href="../styles/main.css" data-gwd-opt="inline">
-    <script type="module" data-gwd-opt="static" src="../components/footer/footer.js"></script>
+    <meta
```

Comment:
Could we avoid extra formatting changes?  It looks your IDE may be auto-formatting but I think it would be easier for readability if purely formatting changes were not included please.